polus-apply-flatfield-plugin/src/main.py

```diff
@@ -1,11 +1,125 @@
 from bfio.bfio import BioReader, BioWriter
-import argparse, logging, subprocess, time, multiprocessing
+import argparse, logging, typing, csv
 import numpy as np
```

Comment:
is this correct `from bfio.bfio import BioReader, BioWriter` ?

Reply:
Yes. In the older version of bfio, this was how to directly import the classes. However, it's not longer necessary.